.browserslistrc

```diff
@@ -0,0 +1 @@
+last 1 version, >2%
```

Comment:
well I wonder if we need to define one here or should we just rely on what the theme has ?

Reply:
It will be used by babel, that is used in webpack and is in current repo(and other tools, that have possibility to use .browserslistrc https://github.com/browserslist/browserslist#browserslist-). We will need to sync all repos together, I've got a PR for cornerstone with more info https://github.com/bigcommerce/cornerstone/pull/1836